src/main/java/org/fundacion/jala/converter/core/VideoConverter.java

```diff
@@ -100,24 +100,24 @@ private String changeResolution() {
      *
      * @throws ConverterException if process is interrupted.
      */
-    private void generateAThumbnail() throws ConverterException, IOException {
+    private void generateAThumbnail() throws ConverterException {
         String name = getOutputFileName().substring(0, getOutputFileName().lastIndexOf("."));
         String startCommand = "ffmpeg -i ";
         String outputCommand = pathOutput + output + "\"" + " -ss 00:00:01 -vframes 1 -s 128x128 "
                 + pathOutput + name + PNG_FORMAT + "\" -y";
         String thumbnailCommand = startCommand + outputCommand;
         System.out.println(thumbnailCommand);
+        try {
         Process process = Runtime.getRuntime().exec("cmd /c " + thumbnailCommand);
         ThreadHandler errorHandler = new ThreadHandler(process.getErrorStream(), "Error Stream");
         errorHandler.start();
         ThreadHandler threadHandler = new ThreadHandler(process.getInputStream(), "Output Stream");
         threadHandler.start();
         LOGGER.info("start");
-        try {
             LOGGER.info("Execute Try");
```

Comment:
I think these 3 loggers are no longer needed: Start, Execute try and finish

Reply:
You are right. I just fixed it